model/src/main/kotlin/OrtResult.kt

```diff
@@ -331,7 +332,21 @@ data class OrtResult(
     /**
      * Return a copy of this [OrtResult] with the [Repository.config] replaced by [config].
      */
-    fun replaceConfig(config: RepositoryConfiguration): OrtResult = copy(repository = repository.copy(config = config))
+    fun replaceConfig(config: RepositoryConfiguration): OrtResult =
```

Comment:
Please also update the documentation of the function to explain how curations contained in `config` are handled.